src/RGUnityBots/Assets/Tests/Runtime/RGActionManagerTests.cs

```diff
@@ -714,7 +715,7 @@ public IEnumerator TestActionManager()
                 }
                 RGActionManager.StopSession();
             }
-            
+
```

Comment:
Lots of whitespace change, I wonder if we all have different formatters setup. We should message out what we have so the PRs don't have this going back and forth between two different formats.

Reply:
Remember ashley's whole thing about auto formatting that basically stopped development for a week while we had to go undo it.. let's not get there again over meaningless whitespaces.  I would suggest you 'hide whitespace' when reviewing in github.